docs/README.rst

```diff
@@ -22,22 +34,38 @@ Debian/Redhat families), Python packages (using pip states) and Ruby gems
 It can also provide basic dependency management on certain other states/packages.
 
 What this formula can't do
-==========================
+--------------------------
 
 This formula is not intended to configure packages, nor setup services or daemons.
 When you need to do that for a package, you should probably be using another
 formula.
 
-.. note::
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
 
-    See the full `Salt Formulas installation and usage instructions
-    <http://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see :ref:`How to contribute <CONTRIBUTING>` for more details.
 
 Available states
-================
+----------------
 
 .. contents::
-    :local:
+   :local:
 
 ``packages``
 ------------
```

Comment:
All of the states mentioned under `Available states` will need to use `^` as the heading symbol, otherwise they're not indented under the table of contents, as shown when viewing the current file:

* https://github.com/saltstack-formulas/packages-formula/blob/6fde6606e742b7cce94a13545341d84e732d692d/docs/README.rst